lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1366,11 +1366,18 @@ class DecomposeAtenMeanOp : public OpRewritePattern<AtenMeanOp> {
     Value input = op.self();
     Value output = op.result();
     BaseTensorType outputTensorType = output.getType().cast<BaseTensorType>();
-    Value sum =
-        rewriter.create<AtenSumOp>(loc, outputTensorType, input, op.dtype());
+    Type newOutputType = outputTensorType.getWithSizesAndDtype(
```

Comment:
nit: I would call this `outputTensorTypeAsF64`, since this type does not represent the actual output type of the `AtenMeanOp`